pkg/server/server.go

```diff
@@ -118,6 +118,7 @@ func setUpEngine(cfg config.ServerConfig, shutdown chan os.Signal) *gin.Engine {
 		gin.Recovery(),
 		gin.Logger(),
 		middleware.Errors(shutdown),
+		middleware.AuthMiddleware(),
```

Comment:
I think we want to comment this out

Reply:
it does skip it: https://github.com/TBD54566975/ssi-service/pull/556/files#diff-6320ecdc95f4d46f3bb6598130be227c00be32852a9774c073715363bae0c374R33 if not set